package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -388,79 +316,147 @@
+    `WaterOrientationalRelaxation` (WOR) indicates
+    "how fast" water molecules are rotating or changing direction.
+    If WOR is very stable we can assume that water molecules are
+    rotating/changing direction very slowly, on the
+    other hand, if WOR decay very fast, we can assume that water molecules are
+    rotating/changing direction very fast. The results are
+    time correlation functions given by:
 
     .. math::
         C_{\hat u}(\tau)=\langle \mathit{P}_2[\mathbf{\hat{u}}(t_0)\cdot\mathbf{\hat{u}}(t_0+\tau)]\rangle
 
-    where :math:`P_2=(3x^2-1)/2` is the second-order Legendre polynomial and :math:`\hat{u}` is
-    a unit vector along HH, OH or dipole vector.
-
+    where :math:`P_2=(3x^2-1)/2` is the second-order Legendre polynomial
+    and :math:`\hat{u}` is a unit vector along HH, OH or dipole vector.
 
     Parameters
     ----------
     universe : Universe
       Universe object
     selection : str
       Selection string for water [‘byres name OH2’].
-    t0 : int
-      frame  where analysis begins
-    tf : int
-      frame where analysis ends
-    dtmax : int
-      Maximum dt size, `dtmax` < `tf` or it will crash.
+
+    Attributes
+    ----------
+    times : numpy.ndarray
+        times for the correlation functions
+    OH : numpy.ndarray
+        correlation function for oxygen-hydrogen vector
+    HH : numpy.ndarray
+        correlation function for oxygen-hydrogen vector
+    dip : numpy.ndarray
+        correlation function for water dipole vector
+
+
+    Examples
+    --------
+
+    .. code-block:: python
+
+        import matplotlib.pyplot as plt
+        import MDAnalysis as mda
+        from MDAnalysis.analysis.waterdynamics import WaterOrientationalRelaxation
+
+        u = mda.Universe(waterPSF, waterDCD)
+        select = "byres name OH2"
+        wor = WaterOrientationalRelaxation(u, select)
+        wor.run(0, 7, 1)
+
+        # Plot the results
+        plt.figure(1, figsize=(18, 6))
+
+        # WOR OH
+        plt.subplot(131)
+        plt.xlabel('time / ps')
+        plt.ylabel('WOR')
+        plt.title('OH orientational relaxation')
+        plt.plot(wor.times, wor.OH, "o-")
+
+        # WOR HH
+        plt.subplot(132)
+        plt.xlabel('time / ps')
+        plt.ylabel('WOR')
+        plt.title('HH orientational relaxation')
+        plt.plot(wor.times, wor.HH, "o-")
+
+        # WOR dip
+        plt.subplot(133)
+        plt.xlabel('time / ps')
+        plt.ylabel('WOR')
+        plt.title('water dipole orientational relaxation')
+        plt.plot(wor.times, wor.dip, "o-")
+
+        plt.show()
 
 
     .. versionadded:: 0.11.0
 
     .. versionchanged:: 1.0.0
        Changed `selection` keyword to `select`
+
+    .. versionchanged:: 2.0.0
+       `t0`, `tf` and `dtmax` are renamed to start, stop, step as attributes
+       of the `run` method providing a consistent behaviour with other
+       modules.
     """
 
-    def __init__(self, universe, select, t0, tf, dtmax, nproc=1):
```

Comment:
Document removal of `nproc` as a versionchanged ("Removed optional ``nproc`` kwarg that was not used.") and in CHANGELOG Changes.

I think we can get away without deprecations for this one.